src/vm/internal/compiler/lex.cc

```diff
@@ -2794,9 +2794,7 @@ void add_predefines() {
   add_quoted_predefine("__OPTIMIZATION__", OPTIMIZE);
 
 /* Backwards Compat */
-#ifndef CDLIB
```

Comment:
Because I couldn't find any other references to thís macro…
And I don't think anyone would use an unchanged CD-lib with either MudOS or FluffOS, something which wouldn't work anyway.
The first 'glitch' would be a master which couldn't be loaded because MudOS is missing some essential applies (master::inaugurate_master at the front).